### PR TITLE
lantern-ui reminds user to upgrade browser if < IE 10

### PR DESCRIPTION
**Pull request by fffw**
Relates to https://github.com/getlantern/lantern/issues/2400 and https://github.com/getlantern/lantern/issues/2261.

### CODE REVIEW

**Comment by fffw** [comment]
Text is only in English. A link to Google Chrome is ok here as Lantern is already running.

Below is what it looks like on IE 8
image

**Comment by fffw** [comment]
Would you like to take a look @atavism ?

File: src/github.com/getlantern/lantern-ui/dist/index.html [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<!DOCTYPE html><html ng-app=app><head><meta charset=utf-8><title>{{ 'LANTERN' | translate }}</title><meta name=viewport content="width=device-width, initial-scale=1.0"><meta name=description content=Lantern><meta name=author content="Team Lantern"><link rel=stylesheet href=style/app.css><script src=js/lib-cb33dd9b.js></script><script src=js/app.js></script><link rel="shortcut icon" href=img/favicon.ico><link rel=icon type=image/png href=img/favicon.png></head></html>
```
Comment (atavism) [inline src/github.com/getlantern/lantern-ui/dist/index.html]: Do we want to delete most of this file?

File: src/github.com/getlantern/lantern-ui/app/_css/app.css [hunk 2]
```diff
@@ -7494,47 +7503,7 @@ body[dir=rtl] #latestDetails dd {
...
-#welcomescreen .configure_system,
-#welcomescreen .now_access {
-    padding: 25px 30px 25px 30px;
-    font-size: 20px;
-    line-height: 25px;
-    color: rgb( 0, 0, 0 );
-    font-weight: bold;
-    text-align: center;
-}
 #contact textarea.report {
```
Comment (atavism) [inline src/github.com/getlantern/lantern-ui/app/_css/app.css]: Same here. Do we want to delete the styling here?

**Comment by fffw** [comment]
I just modified `sass/_general.sass` and `app/index.html`, all other files are generated or copied automatically. `app.css` is generated from sass files, and `dist/index.html` is copied from `app/index.html`.

I think these two files might not be properly updated previously.

**Comment by atavism** [comment]
Ok, thanks! Should be fine to merge then. Closing.

**Pull request merged**